### ISSUE TITLE
Fix regression error for `@import`

### ISSUE THREAD

**Issue by MartijnCuppens** [stylelint/stylelint#4332]
> Clearly describe the bug

We are trying to upgrade stylint from `10.1.0` to `11.0.0`, but there seems to be an issue with `.sass` files. `@import` statements always seem to fail.

> Steps to reproduce:

- Checkout the `xmr-update-stylelint` of https://github.com/twbs/rfs
https://github.com/twbs/rfs/pull/168
- run `npm install`
- run `npm run stylelint`

or lint a `.sass` file with an `@import` statement. 

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint "**/*.{less,sass,scss}"`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Sass.

> What actually happened (e.g. what warnings or errors did you get)?

```shell
TypeError: Cannot read property 'start' of undefined
    at Rule.positionBy (/Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/node.js:488:27)
    at module.exports (/Users/martijn.cuppens/workspace/rfs/node_modules/stylelint/lib/utils/report.js:49:34)
    at check (/Users/martijn.cuppens/workspace/rfs/node_modules/stylelint/lib/rules/block-no-empty/index.js:70:7)
    at /Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/container.js:239:18
    at /Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/container.js:135:18
    at Root.each (/Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/container.js:101:16)
    at Root.walk (/Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/container.js:131:17)
    at Root.walkRules (/Users/martijn.cuppens/workspace/rfs/node_modules/postcss/lib/container.js:237:19)
    at /Users/martijn.cuppens/workspace/rfs/node_modules/stylelint/lib/rules/block-no-empty/index.js:43:10
    at performRules.push.Promise.all.postcssRoots.map.postcssRoot (/Users/martijn.cuppens/workspace/rfs/node_modules/stylelint/lib/lintSource.js:247:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! rfs@9.0.0 stylelint: `stylelint "**/*.{less,sass,scss}" --cache --cache-location node_modules/.cache/stylelint`
npm ERR! Exit status 1
```

Possibly related to https://github.com/stylelint/stylelint/issues/4109.

CC @XhmikosR

**Comment by hudochenkov** [comment]
Thanks for the report and for using the template.

Consider contributing a fix.

**Comment by XhmikosR** [comment]
I updated the patch to use Stylelint 11.1.1, the error is still there BTW.

**Comment by MartijnCuppens** [comment]
It was introduced by https://github.com/stylelint/stylelint/pull/4148, I'll check what's going on upstream.

**Comment by Srilalitha1310** [comment]
Me and [@niranjanib](https://github.com/niranjanib) .. Are looking into this issue.

**Comment by tsubonet** [comment]
I have similar errors. In my case, there were errors where the media query was written. 
It worked well by importing @media as a separate file.

style.sass
```
@import “../path/mixins"

.hoge
   media()
     margin: 10px
```

mixins.sass
```
=media()
  @media all and (max-windth: 600px)
    @content
```

**Comment by joehannes** [comment]
ping, any news on this??

**Comment by danilo-znamerovszkij** [comment]
Still seems to happen..

**Comment by jeddy3** [comment]
The [`syntax` option was removed](https://github.com/stylelint/stylelint/issues/5289) for the [14.0.0 release](https://github.com/stylelint/stylelint/issues/5205). (See the [migration guide](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-14.md#syntax-option-and-automatic-inferral-of-syntax).)

Please consider opening an issue upstream and fixing the issue there as the syntax is no longer bundled by Stylelint.
